cli/flox/src/commands/environment.rs

```diff
@@ -1404,12 +1437,17 @@ pub struct Push {
 }
 
 impl Push {
+    #[instrument(name = "push", skip_all)]
     pub async fn handle(self, mut flox: Flox) -> Result<()> {
         subcommand_metric!("push");
 
         // Ensure the user is logged in for the following remote operations
         ensure_floxhub_token(&mut flox).await?;
 
+        // Start a span that doesn't include authentication
```

Comment:
.. or authenticate in a subspan?
What is the reason the have another span for "post-auth" stuff?

Reply:
I did it this way so that you can get duration information directly without needing to subtract out the duration of an `auth` span.